draft-ietf-rats-corim.md

```diff
@@ -1462,7 +1626,7 @@ translated into a separate evidence object.
 The Verifier SHALL translate each field in the TcbInfo into a field in the
 created endorsed-triple-record
 
-- The TcbInfo `type` field SHALL be copied to the field named `environment-map / class / class-id`
+- The TcbInfo `type` field SHALL be copied to the field named `environment-map / class / class-id` and tagged with tag #6.111
```

Comment:
The TcbInfo TYPE is an OCTET STRING that can contain whatever form that `class-id` can take. I believe it supports tagged UUID and tagged int in addition to tagged OID. The Verifier reads the RV to know how to translate the Evidence OCTET STRING. This poses a problem for Verifiers that copy the Evidence into ACS without knowing the RV context. Possibly, they need a dirty bit that tells whether the schema translation has been applied?

Reply:
Created issue #138